otel-integration/k8s-helm/values.yaml

```diff
@@ -0,0 +1,470 @@
+      #   timeout: 2s
+      #   override: false
+      #   gcp:
+      #     resource_attributes:
+      #       cloud.region:
+      #         enabled: true
+      #       cloud.availability_zone:
+      #         enabled: true
+      #   ec2:
+      #     resource_attributes:
+      #       cloud.region:
+      #         enabled: true
+      #       cloud.availability_zone:
+      #         enabled: true
+      metricstransform:
+        transforms:
+          include: .*
+          match_type: regexp
+          action: update
+          operations:
+            - action: add_label
+              new_label: k8s.cluster.name
+              new_value: "{{ .Values.global.clusterName }}"
+            - action: add_label
+              new_label: cx.otel_integration.name
+              new_value: "{{ .Chart.Name }}"
+            - action: add_label
+              new_label: cx.otel_integration.version
+              new_value: "{{ .Chart.Version }}"
+      k8sattributes:
+        filter:
+          node_from_env_var: KUBE_NODE_NAME     
+        extract:
+          metadata:
+            - "k8s.namespace.name"
+            - "k8s.deployment.name"
+            - "k8s.statefulset.name"
+            - "k8s.daemonset.name"
+            - "k8s.cronjob.name"
+            - "k8s.job.name"
+            - "k8s.pod.name"
+            - "k8s.node.name"
+      spanmetrics:
+        metrics_exporter: coralogix
+        dimensions:
+          - name: "k8s.deployment.name"
+          - name: "k8s.statefulset.name"
+          - name: "k8s.daemonset.name"
+          - name: "k8s.cronjob.name"
+          - name: "k8s.job.name"
+          - name: "k8s.container.name"
+          - name: "k8s.node.name"
+          - name: "k8s.namespace.name" 
+      memory_limiter: null # Will get the k8s resource limits
+
+    exporters:
+      coralogix:
+        timeout: "30s"
+        private_key: "${CORALOGIX_PRIVATE_KEY}"
+        domain: "{{ .Values.global.domain }}"
+        application_name: "{{ .Values.global.defaultApplicationName }}"
+        subsystem_name: "{{ .Values.global.defaultSubsystemName }}"
+        application_name_attributes:
+          - "k8s.namespace.name"
+          - "service.namespace"
+        subsystem_name_attributes:
+          - "k8s.deployment.name"
+          - "k8s.statefulset.name"
+          - "k8s.daemonset.name"
+          - "k8s.cronjob.name"
+          - "k8s.job.name"
+          - "k8s.container.name"
+          - "k8s.node.name"
+          - "service.name"
+
+    service:
+      telemetry:
+        logs:
+          level: "{{ .Values.global.logLevel }}"
+          encoding: json
+        metrics:
+          address: ${MY_POD_IP}:8888
+      extensions:
+      - zpages
+      - pprof
+      - health_check
+      - memory_ballast
+      pipelines:
+        metrics:
+          exporters:
+            - coralogix
+          processors:
+            - k8sattributes
+            - resourcedetection/env
+            # - resourcedetection/region
```

Comment:
Should we remove the comment?

Reply:
Good eyes. Looks like the comment got lost, I added a TODO here - we first need to upgrade the collector, which is blocked by https://coralogix.atlassian.net/browse/ES-51, since we need latest feature for some detectors and configs to work. I'll re-add the comment or ideally try to unblock before this is merge.